lib/galaxy/webapps/galaxy/api/datatypes.py

```diff
@@ -9,14 +9,10 @@
     Union,
 )
 
-from fastapi import (
-    Depends,
-    Query,
-)
+from fastapi import Query
 from fastapi_utils.cbv import cbv
 from fastapi_utils.inferring_router import InferringRouter as APIRouter
```

Comment:
I think here those imports and the `APIRouter` can be replaced by the new `Router` :)

Reply:
Rebased with a fix for this, thanks for the catch!